### ISSUE TITLE
Dragon Caves - Surface tunnel network

### ISSUE THREAD

**Issue by zth404** [AlexModGuy/Ice-and-Fire-Suggestions#867]
A series of tunnels and what not from the surface leading to the big underground dragon chambers with the Tier 4 and 5 dragons.

**Comment by sj134107** [comment]
Yeah make it to where the dragons have a way out to go hunt, or whatnot, seems more realistic
